tests/transactional/install_updates.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Install Update repos in transactional server
+# Maintainer: qac team <qa-c@suse.de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use qam;
+use transactional;
+
+sub run {
+    my ($self) = @_;
+    select_console 'root-console';
+    add_test_repositories;
+    record_info 'Updates', script_output 'zypper lu';
```

Comment:
There the implicit function call syntax makes it harder to read IMO. What about

```suggestion
    record_info 'Updates', script_output('zypper lu');
```

Reply:
Doesn't this make some `svirt` backends more prone to serial terminal errors? (cc: @mloviska

Reply:
The problem was calling record_info with only 1 argument if I remember correctly.